spec/mailers/notification_mailer_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+RSpec.describe NotificationMailer, type: :mailer do
+  describe 'release_email' do
+    let(:user) { mock_model User, first_name: 'Ignas', last_name: 'Panero Armoska', email: 'ignaspan@gmail.com' }
+    let(:fake_assignment) { mock_model Assignment, short_identifier: 'A2'}
+    let(:grouping) { mock_model Grouping, assignment: fake_assignment}
+    let(:mail) { described_class.with(user: user, grouping: grouping).release_email.deliver_now }
+
+    it 'renders the subject' do
+      expect(mail.subject).to eq('MarkUs Notification NOT SURE FOR COURSE Your marks for A2 have released!')
```

Comment:
`Rails.configuration` should still be accessible here.

Also instead of `A2` it's better to do `fake_assignment.short_identifier`.